html.go

```diff
@@ -901,31 +902,10 @@ func doubleSpace(out *bytes.Buffer) {
 }
 
 func isRelativeLink(link []byte) (yes bool) {
-	// a tag begin with '#'
-	if link[0] == '#' {
-		return true
+	u, err := url.Parse(string(link[:]))
```

Comment:
I got this hint in stackoverflow (details [here](http://stackoverflow.com/questions/14230145/what-is-the-best-way-to-convert-byte-array-to-string)). The user ferhat elmas [commented](http://stackoverflow.com/questions/14230145/what-is-the-best-way-to-convert-byte-array-to-string#comment55642294_18615786) about this issue:
> a string is in effect a read-only slice of bytes. You can't convert byte array directly to string so first slice then string".

If you try a direct convert, you will get an error, like this example: 
http://play.golang.org/p/Wdr1_sOzMH